osi_environment.proto

```diff
@@ -324,4 +233,102 @@ message EnvironmentalConditions
         //
         optional uint32 seconds_since_midnight = 1;
     }
+
+    //
+    // \brief Definition of the ambient illumination
+    //
+    // \note Ambient light is any light in the vehicle's environment that is not
+    // emitted by the vehicle itself. It can include sun/moon light, light from
+    // other cars, street lights etc.
+    //
+    message AmbientIllumination
+    {
```

Comment:
You're missing the actual field value which uses the enum from this message.

For example `AmbientIlluminationLevel level = 1;`.

Reply:
You are absolutely right.  I haven't added a field yet, because how to name the ENUM field and how to name the field for the numerical value range?

`AmbientIlluminationLevel level = 1`
`double intensity = 2`